### PR TITLE
Update: changelog for 0.7.1

### PR DESCRIPTION
**Pull request by glx22**
While writing documentation for stations, we made some small changes to the syntax.
Let's release a version matching the documentation.

### CODE REVIEW

File: docs/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+- Change: allow any expression for station (purchase_)prepare_layout
+- Fix: Station animation triggers every 250 ticks, not 256. (#266)
+- Change: replace nearby_tile_platform_type with nearby_tile_tile_type. (#265)
+- Change: disabled_platforms/length is now bitmask(1-8) (#264)
+- Change: use an array of cargoes for cargo_random_triggers (#263)
+- Fix 16eb0035: disabled_platforms and disabled_length properties used a wrong number
+
+Other changes and fixes:
+- Fix #256: Objects have a 'colour' in var 47. (#267)
+- Cleanup: burn tabs into spaces.
```
Comment (michicc) [inline docs/changelog.txt:16]: burn? 😄

File: docs/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,21 @@
+0.7.1 (2022-11-30)
+------------------------------------------------------------------------
+While writing documentation for stations, we made some small changes to the syntax.git rm
```
Comment (LordAro) [inline docs/changelog.txt:3]: git rm

**Comment by glx22** [comment]
Fixed on phone, will squash anyway on merge.

**Review by michicc — APPROVED** [review]

**Comment by glx22** [comment]
Release workflow is untested, I hope it still works.

**Review by LordAro — APPROVED** [review]
Easy way to find out

**Pull request merged**